docs/user-guide/ze-usage-tips.md

```diff
@@ -12,7 +12,17 @@ Zowe Explorer supports syntax highlighting for data sets and USS files. To enabl
 
 ## Configure the detected language of a file or data set
 
-Configure Visual Studio Code to use a specific language for a particular file extension type. This prevents the language for a file or data set opened in Zowe Explorer from being detected incorrectly. To set file associations, see [Add a file extension to a language](https://code.visualstudio.com/docs/languages/overview#_add-a-file-extension-to-a-language).
+Configure the `file.associations` setting in Visual Studio Code to use a specific language for a particular file extension type. This prevents the language for a file or data set opened in Zowe Explorer from being detected incorrectly.
+
+ To set file associations that work for sequential data sets as well as partitioned data set members, use wildcards with the language identifier in the format `**/*LANGUAGE*{,/*}`:
+
+```
+    "files.associations": {
+        "**/*COBOL*{,/*}": "cobol"
+    }
+```
+
+In the example above, Zowe Explorer uses wild cards to find matches of the configured language (`COBOL`) in the file paths of sequential data sets (for example, `/lpar.zosmf/TEST.COBOL.PS`) and PDS members (for example, `/lpar.zosmf/TEST.COBOL.PDS/MEMBER`).
```

Comment:
@t1m0thyj @traeok: Should we say here that the instructions for V2 are different and either (1) link to them, or (2) write them out here?

Reply:
In V2 the paths were structured differently, such that the data set name was always contained in the filename (or last segment of the file path). Paths for sequential data sets were formatted the same way (`/lpar.zosmf/TEST.COBOL.PS`) and for PDS members were different (`/lpar.zosmf/TEST.COBOL.PDS(MEMBER)`).

Because of this the wildcard pattern to match `COBOL` was significantly simpler - just `*COBOL*` would achieve the same effect in V2 as `**/*COBOL*{,/*}` in V3.

Reply:
Although you can technically use a shorter glob for v2, could we recommend  the same one for both v2 and v3?

As far as I can tell, this glob will behave the same way as the shorter one. And this way, users can keep their file associations with this glob pattern and will be able to leverage them without issue if they jump from v2 -> v3 (e.g., once v2 is EOL)

Reply:
I like this idea - although a simpler glob works in V2, the same glob will work in both V2 and V3, so we can probably avoid needless complexity by keeping the docs the same for both versions 😋

Reply:
ok, I won't add any further doc then re: v2 -- thank you